code/game/objects/items/ego_weapons/aleph.dm

```diff
@@ -342,21 +337,21 @@
 			Looking at the E.G.O for more than 3 seconds will make you sick."
 	special = "Using it in hand will activate its special ability. To perform this attack - click on a distant target."
 	icon_state = "censored"
-	force = 85
+	force = 70	//there's a focus on the ranged atttack here.
```

Comment:
There is no focus on ranged... It has high cooldown and subpar damage.

Reply:
I had to balance it for 100 PE somehow.
It being 80 with a free 360 damage attack is a little too much to be on par with Mimicry and Smile